Management/Clients/AirQualityDataClient.cs

```diff
@@ -26,13 +31,17 @@ public AirQualityDataClient(IConfiguration configuration)
             _configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
         }
 
-        public async Task<AirQualityCityResponse> GetAirQualityAsync(string city, string state, string countryCode, CancellationToken cancellationToken)
+        public async Task<AirQualityCityResponse> GetAirQualityAsync(
+            City city,
+            State state,
+            CountryCode countryCode,
+            CancellationToken cancellationToken)
         {
             var request = new RestRequest("v2/city", Method.GET);
             request
-                .AddQueryParameter("city", city)
-                .AddQueryParameter("state", state)
-                .AddQueryParameter("country", countryCode)
+                .AddQueryParameter("city", city.Value)
+                .AddQueryParameter("state", state.Value)
+                .AddQueryParameter("country", countryCode == CountryCode.US ? "USA" : "CANADA")
```

Comment:
Can be improved here, but for now just keep it simple.

Later on we need to figure out what other 2 api we used the country code standard they are using. ISO2 OR ISO3.

Reply:
So for the COVID-19 tracking project, they currently only support US's COVID-19 data. Thus, I don't think they would use any country code standard. Maybe we could switch to another covid API that has data from worldwide for future expansion purposes. As for AccuWeather, here is the link: https://developer.accuweather.com/countries-by-region. They are using Alpha-2 codes for countries. We will need to take care of two things in the future: 1. do all three APIs support the requested country/state/city? 2. process to get consistent names for state/cities across three APIs (NY vs. NYC vs. New York vs. New York City)